tests/functional/test_accounts.py

```diff
@@ -18,7 +18,7 @@ def test_submit_email_form_without_xhr_returns_full_html_page(self, app):
 
         res = email_form.submit().follow()
 
-        assert res.body.startswith('<!DOCTYPE html>')
+        assert res.text.startswith('<!DOCTYPE html>')
```

Comment:
This is needed because these tests now fetch the Activity Pages version of the site, which has some non-ASCII characters in parts of the template (eg. the navbar).